src/test/bare-module-worker_test.ts

```diff
@@ -1039,6 +1039,88 @@ suite('bare module worker', () => {
     await checkTransform(files, expected, {}, cdn);
   });
 
+  test('prefers "import" + "browser" condition exports over default', async () => {
+    const files: SampleFile[] = [
+      {
+        name: 'index.js',
+        content: `
+          import 'foo';
+          import 'foo/bar.js';
+        `,
+      },
+    ];
+
+    const cdn: CdnData = {
+      foo: {
+        versions: {
+          '1.0.0': {
+            files: {
+              'browser/index.js': {
+                content: 'BROWSER',
+              },
+              'browser/bar.js': {
+                content: 'BROWSER',
+              },
+              'default/index.js': {
+                content: 'DEFAULT',
+              },
+              'default/bar.js': {
+                content: 'DEFAULT',
+              },
+              'package.json': {
+                content: `{
+                  "exports": {
+                    ".": {
+                      "import": {
+                        "browser": "./browser/index.js"
```

Comment:
Worth adding a test matching the structure for the specific example?

https://unpkg.com/@floating-ui/dom@0.5.0/package.json

Example containing: `import.browser.default : <filename>`

Reply:
I think the test coverage is already ok, there are some other tests in here handling defaults. We also use the actual implementation from Node itself so the risk is fairly low to begin with.